zebedee-cms/src/main/java/com/github/onsdigital/zebedee/util/slack/Notifier.java

```diff
@@ -0,0 +1,17 @@
+package com.github.onsdigital.zebedee.util.slack;
+
+import com.github.onsdigital.zebedee.model.Collection;
+
+
+/**
+ * Notifier is a generic interface for sending notifications to external systems such as Slack
+ */
+public interface Notifier {
```

Comment:
I think the SlackNotification util is used a number of times in the Root class, possibly scope creep but it feels like if we are defining an interface to wrap around this util we should probably include all the methods being used in the Root class. For consistency if nothing else.